src/others/helpers/use_query.ts

```diff
@@ -0,0 +1,63 @@
+import { useMemo, useState } from "react";
```

Comment:
**nitpick**

Please, let's stick to kebab-case in naming .ts files.

Reply:
Good catch!